web/src/components/storage/dasd/StatusFilter.tsx

```diff
@@ -31,18 +31,18 @@ import {
   SelectProps,
 } from "@patternfly/react-core";
 import Text from "~/components/core/Text";
-import { _ } from "~/i18n";
+import { N_, _ } from "~/i18n";
 
 type StatusFilterProps = {
   value: string;
   onChange: SelectProps["onSelect"];
 };
 
 const options = {
-  all: _("all"),
-  active: _("active"),
-  read_only: _("read_only"),
-  offline: _("offline"),
+  all: N_("all"),
```

Comment:
NP:
this does fix the translation code but the translators may get it wrong because of the missing context,
often the grammatical gender gets mismatched.
And I see these are applied in different contexts... where the subject is always "devices"?